td.server/test/config/parsers.config_spec.js

```diff
@@ -38,15 +40,15 @@ describe('request parser tests', () => {
         const body = {id: id, collection: collection, nested: nested};
         
         app.post('/', function(req, res) {
-            expect(req.body["collection[0]"]).to.eq(collection[0]);
-            expect(req.body["collection[1]"]).to.eq(collection[1]);
-            expect(req.body["collection[2]"]).to.eq(collection[2]);
-            expect(req.body["nested[n1]"]).to.eq(nested.n1);
-            expect(req.body["nested[n2]"]).to.eq(nested.n2);
+            expect(req.body.collection[0]).to.eq(collection[0]);
```

Comment:
Removing the bodyParser dependency and using the built-in express parser changed how this works a bit.  Using dot-notation works just fine.

I searched the codebase and didn't find any instances of `req.body[`, so I believe this to be a non-breaking change.